### PR TITLE
Fix hostNodes example

### PR DESCRIPTION
**Pull request by vkrol**
This example is a little confusing. I think it's a more correct version of it.

### CODE REVIEW

File: content/en/all/react-testing-2-jest-and-enzyme.md [hunk 1]
```diff
@@ -694,9 +694,9 @@ wrapper.find('[data-testid="pizzaForm-submit"]').simulate('click');
 
 This won’t work because `find()` returns two nodes: one for the `Button` React component, and one for the `button` HTML element, because the component tree would look like this:
 
-```html
-<button data-testid="pizzaForm-submit">
-  <button data-testid="pizzaForm-submit">Cook pizza!</button>
+```jsx
+<Button data-testid="pizzaForm-submit">
+  <button className="Button" data-testid="pizzaForm-submit">Cook pizza!</button>
 </button>
```
Comment (sapegin) [inline content/en/all/react-testing-2-jest-and-enzyme.md]:
```suggestion
</Button>
```

File: content/en/all/react-testing-2-jest-and-enzyme.md [hunk 2]
```diff
@@ -680,9 +680,9 @@ const Button = props => <button className="Button" {...props} />;
 
 You have a form:
 
-```html
+```jsx
 <form>
-  <button data-testid="pizzaForm-submit">Cook pizza!</button>
+  <Button data-testid="pizzaForm-submit">Cook pizza!</button>
```
Comment (sapegin) [inline content/en/all/react-testing-2-jest-and-enzyme.md]:
```suggestion
  <Button data-testid="pizzaForm-submit">Cook pizza!</Button>
```

**Comment by vkrol** [comment]
No problem. Done.

**Comment by sapegin** [comment]
Merging, thanks! :shipit:

**Pull request merged**